### PR TITLE
Feat(sharepoint): add new task tab

### PR DESCRIPTION
**Pull request by rfougier**

### CODE REVIEW

File: src/sharepoint/task/TASK.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                              data-ng-bind="tr('privateDatabase_tabs_tasks_status_' + task.status)"
+                              data-ng-class="{
+                                'label-success' : task.status === 'done',
+                                'label-info': task.status === 'doing' || task.status === 'todo',
+                                'label-danger': task.status === 'error',
+                                'table-warning': task.status === 'cancelled'
+                              }"
+                              data-ng-if="task.status"></span>
+                    </td>
+                    <td></td>
```
Comment (antleblanc) [inline src/sharepoint/task/TASK.html]: -`<td></td>`

File: src/sharepoint/task/TASK.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                    <td class="loader text-center" colspan="5"></td>
+                </tr>
+            </tbody>
+
+            <tbody data-ng-if="!taskCtrl.loaders.tasks && taskCtrl.tasksIds.length > 0">
+                <tr data-ng-repeat="task in taskCtrl.taskDetails track by $index">
+                    <th scope="row" data-ng-bind="::task.function"></th>
+                    <td data-ng-bind="::task.todoDate | date:'medium'"></td>
+                    <td data-ng-bind="::task.finishDate | date:'medium'"></td>
+                    <td class="text-center">
```
Comment (antleblanc) [inline src/sharepoint/task/TASK.html]: +`colspan="2"`

File: src/sharepoint/task/sharepoint-task.controller.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            this.tasksIds = null;
+
+            return this.sharepointService.getTasks(this.$stateParams.exchangeId)
+                .then((ids) => {
+                    this.tasksIds = ids;
+                    if (_.isArray(ids) && !_.isEmpty(ids)) {
+                        this.hasResult = true;
+                    }
+                })
+                .finally(() => {
```
Comment (antleblanc) [inline src/sharepoint/task/sharepoint-task.controller.js]: No reason to `catch`?

File: src/sharepoint/sharepoint-tabs.controller.js [hunk 4]
```diff
@@ -13,7 +13,7 @@ angular
             this.defaultTab = "INFORMATION";
 
             this.$scope.toKebabCase = _.kebabCase;
-            this.$scope.tabs = ["INFORMATION", "ACCOUNT"];
+            this.$scope.tabs = ["INFORMATION", "ACCOUNT", "TASK"];
```
Comment (FredericEspiau) [inline src/sharepoint/sharepoint-tabs.controller.js]: `TASKS` ?

Comment (rfougier) [reply src/sharepoint/sharepoint-tabs.controller.js]: Nope

**Pull request merged**